templates/partials/contents.html

```diff
@@ -1,6 +1,10 @@
 {% for item in contents %}
   {%- if item.title -%}
-    <strong class="u-mb-s">{{item.title}}</strong>
+    {%- if title_type == 'interstitial' -%}
```

Comment:
Can't the block type be checked here rather than having to set another variable?